challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |word| word.capitalize}
```

Comment:
Once folks work with Ruby for a while, they tend to use a shortcut for this kind of code. It can also be written as:

```ruby
input.map(&:capitalize)
```

It's called "symbod to proc" or `Symbol#to_proc`. Understanding how it works is kinda advanced, but the idea is simple: Call a method named "capitalize" on every element of "input".